pygments/lexers/mcfunction.py

```diff
@@ -175,13 +172,13 @@ class MCFunctionLexer(RegexLexer):
         ],
         "comments.block.normal": [
             include("comments.block.special"),
-            (r"\S+?", Comment.Multiline),
+            (r".+", Comment.Multiline),
```

Comment:
Hm, now the "comments.block.special" and "whitespace" includes will not match within the comment.

Usually you can handle it like you did in strings: `[everything that doesn't start a special token]+`

Reply:
Yea, I was thinking about how to get the comment tokens to capture more than just a word, but I forgot to revert it properly (took a lunch break 😅).